src/components/App/App.tsx

```diff
@@ -1204,23 +1232,23 @@ export default class App extends React.Component<null, AppState> {
                       />
                     )}
                     {false && (
-                      <SearchComponent
+                      <YouTubeSearchBar
```

Comment:
I don't think we should rename this. . . it's confusing when it's not searching YouTube